code/game/objects/items/cards_ids.dm

```diff
@@ -195,7 +195,8 @@
 		to_chat(user, "Alt-Click your ID in-hand to pull money from your account in the form of holochips.")
 		to_chat(user, "You can insert credits into your account by pressing holochips against the ID.")
 		to_chat(user, "If you lose this ID card, you can reclaim your account by using a blank ID card inhand and punching in the account ID.")

```

Comment:
while you're here this message is incorrect now, would you mind fixing it?

Reply:
wait it is? What do we do now

Reply:
oh right i see

Reply:
Alt-Click, no?